File: Python/Kendaraan.py
class Kendaraan:

    def __init__(self, jenis_bahan_bakar=None, kapasitas_penumpang=None,
                tahun_produksi=None, status_mesin=None):
        self.__jenis_bahan_bakar = jenis_bahan_bakar
        self.__kapasitas_penumpang = kapasitas_penumpang
        self.__tahun_produksi = tahun_produksi
        self.__status_mesin = status_mesin

    # getter setter
    def get_jenis_bahan_bakar(self):
        return self.__jenis_bahan_bakar

    def set_jenis_bahan_bakar(self, jenis_bahan_bakar):
        self.__jenis_bahan_bakar = jenis_bahan_bakar

    def get_kapasitas_penumpang(self):
        return self.__kapasitas_penumpang

    def set_kapasitas_penumpang(self, kapasitas_penumpang):
        self.__kapasitas_penumpang = kapasitas_penumpang

    def get_tahun_produksi(self):
        return self.__tahun_produksi

    def set_tahun_produksi(self, tahun_produksi):
        self.__tahun_produksi = tahun_produksi

    def get_status_mesin(self):
        return self.__status_mesin

    def set_status_mesin(self, status_mesin):
        self.__status_mesin = status_mesin

File: Php/Kendaraan.php
<?php

class Kendaraan {

    private $jenisBahanBakar;
    private $kapasitasPenumpang;
    private $tahunProduksi;
    private $statusMesin;

    public function __construct($jenisBahanBakar = null, $kapasitasPenumpang = null, $tahunProduksi = null, $statusMesin = null) {
        $this->jenisBahanBakar = $jenisBahanBakar;
        $this->kapasitasPenumpang = $kapasitasPenumpang;
        $this->tahunProduksi = $tahunProduksi;
        $this->statusMesin = $statusMesin;
    }

    // getter setter
    public function getJenisBahanBakar() {
        return $this->jenisBahanBakar;
    }

    public function setJenisBahanBakar($jenisBahanBakar) {
        $this->jenisBahanBakar = $jenisBahanBakar;
    }

    public function getKapasitasPenumpang() {
        return $this->kapasitasPenumpang;
    }

    public function setKapasitasPenumpang($kapasitasPenumpang) {
        $this->kapasitasPenumpang = $kapasitasPenumpang;
    }

    public function getTahunProduksi() {
        return $this->tahunProduksi;
    }

    public function setTahunProduksi($tahunProduksi) {
        $this->tahunProduksi = $tahunProduksi;
    }

    public function getStatusMesin() {
        return $this->statusMesin;
    }

    public function setStatusMesin($statusMesin) {
        $this->statusMesin = $statusMesin;
    }
}